dev/aarch64_opt/src/Makefile

```diff
@@ -12,12 +12,19 @@ TARGET_MICROARCH=Arm_Cortex_A55
 
 SLOTHY_FLAGS=-c sw_pipelining.enabled=true \
              -c inputs_are_outputs \
-             -c reserved_regs="[x18--x30,sp]" \
              -c sw_pipelining.minimize_overlapping=False \
              -c sw_pipelining.allow_post \
              -c variable_size \
              -c constraints.stalls_first_attempt=64
 
+# For some units, we allow SLOTHY to use ALL the V/Q registers. These units MUST
+# save and restore v8-v15 as part of their entry/exit. Registers x18-x30 and sp
+# must always be avoided to respect the ABI.
```

Comment:
Not necessarily, X19-X30 could be stashed, too, but the SIMD-heavy routines we're optimizing here don't need them. It's different for Keccak, for example, where you really every register.

Reply:
I think I know what you mean... let me try to draft something.